hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientGrpc.java

```
@@ -283,6 +284,13 @@ public ContainerCommandResponseProto sendCommand(
     return responseProtoHashMap;
   }
 
+  // TODO: We need separate XceiverClientRatis and XceiverClientGrpc instances
+  //  and remove XceiverClientSpi
+  @Override
+  public DataStreamApi getDataStreamApi() {
+    return null;
+  }
```

Its better to throw an exception here.